committee.rst

```diff
@@ -0,0 +1,152 @@
+GHC Steering Committee Bylaws
+=============================
+
+This document describes the GHC Steering Committee and the
+bylaws by which it operates.
+
+* `Current committee membership <https://github.com/ghc-proposals/ghc-proposals/#who-is-the-committee>`_
+
+Committee Composition
+---------------------
+
+The committee is formed of roughly 10 members of the Haskell community
+who wish to volunteer for this service. It is our aim that this committee
+be diverse; by representing different viewpoints, we will make decisions
+that benefit larger segments of our community.
+
+Specifically, we ideally would be able to represent the following
```

Comment:
Specifically, we would like the committee to represent at least the following constituencies: